styles/global.scss

```diff
@@ -8,9 +8,18 @@ main {
   //without this, footer image is above main contents
   // block hover, click events
   z-index: 100;
+  margin: 0;
+  width: 100%;
+  height: 100%;
```

Comment:
Do we really need this? Does it fix something?

Reply:
removed for now, seems fine